Ds3Examples/BulkPutWithStreamStrategyExample.cs

```diff
@@ -0,0 +1,96 @@
+﻿/*
+ * ******************************************************************************
+ *   Copyright 2014 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+using Ds3;
+using Ds3.Helpers;
+using Ds3.Helpers.Strategys;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+
+namespace Ds3Examples
+{
+    public class BulkPutWithStreamStrategyExample
+    {
+        private static readonly TraceSwitch ClientSwitch = new TraceSwitch("clientSwitch", "Controls tracing for example client");
+
+        public static void Main()
+        {
+            // Configure and build the core client.
+            var client = new Ds3Builder(
+                ConfigurationManager.AppSettings["Ds3Endpoint"],
+                new Credentials(
+                    ConfigurationManager.AppSettings["Ds3AccessKey"],
+                    ConfigurationManager.AppSettings["Ds3SecretKey"]
+                )
+            ).Build();
+
+            // Set up the high-level abstractions.
+            IDs3ClientHelpers helpers = new Ds3ClientHelpers(client);
+
+            const string bucket = "bucket-name";
```

Comment:
i prefer to use a bucket name like "put-bulk-with-stream-test" so its easier to follow in logs and look at if it ever regresses